ui/packages/tidb-dashboard-for-dbaas/src/apps/Monitoring/context.ts

```diff
@@ -62,10 +62,12 @@ export const ctx: (globalConfig: IGlobalConfig) => IMonitoringContext = (
   ds: new DataSource(globalConfig),
   cfg: {
     getMetricsQueries: (pdVersion: string | undefined) =>
-      getMonitoringItems(pdVersion),
+      getMonitoringItems(pdVersion, globalConfig.clusterInfo.deployType),
     timeRangeSelector: {
       recent_seconds: RECENT_SECONDS,
       customAbsoluteRangePicker: true
-    }
+    },
+    metricsWithoutCategory:
+      globalConfig.clusterInfo.deployType === 'Dedicated' ? false : true
```

Comment:
`globalConfig.clusterInfo.deployType === 'Dedicated'` is fine, it returns boolean